packages/ui/src/services/component-config/component-config.ts

```diff
@@ -1,11 +1,12 @@
 import { useLocalConfig } from '../../components/va-config/VaConfig'
 import { useGlobalConfig } from '../global-config/global-config'
-import { computed, DefineComponent, ref } from 'vue'
+import { ComponentInternalInstance, computed, DefineComponent } from 'vue'
 
 export type Props = { [propName: string]: any }
 export type ComponentConfig = { [componentName: string]: Props }
+export type ComponentPreset = { [componentName: string]: { [presetName: string]: Props } }
```

Comment:
```suggestion
export type ComponentPreset = { [presetName: string]: { [componentName: string]: Props | Props } }
```
in this case we can remove `componentsAll` from `GlobalConfig` and add a key `default` (as example) to the `componentsPresets`. And `componentsPresets` could be like this:
```ts
componentsPresets = {
  default: {
    color: 'green' // for all components
    VaButton: {
      color: 'lightgreen',
      class: 'with-🐸',
    },
  },
  customPreset: {
    color: 'orange' // for all components
    VaButton: {
      class: 'with-custom-🐸'
    },
  },
}
```

Reply:
@RVitaly1978
Here is related discussion in Slack:
https://dream-support.slack.com/archives/CG75YRY1Y/p1654616615000269

In two words:
We didn't decide it yet because there are some pros and cons to this.